react/src/components/organisms/HeaderSlim/index.js

```diff
@@ -4,36 +4,39 @@
  * @requires module:@massds/mayflower-assets/scss/03-organisms/header-slim
  */
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const HeaderSlim = (props) => (
+const HeaderSlim = ({
+  SkipNav,
+  SiteLogo,
+  MainNav,
+  UtilityNav
+}) => (
   <div className="ma__header_slim">
-    <div className="ma__header_slim__banner" />
+    {SkipNav && <SkipNav />}
+    <div className="ma__header_slim__utility">
+      <div className="ma__header_slim__utility-container ma__container">
+        {UtilityNav && <UtilityNav />}
+      </div>
+    </div>
     <header className="ma__header_slim__header" id="header">
-      <div className="ma__header_slim__container ma__container">
-        <div className="ma__header_slim__logo">{props.siteLogo}</div>
-        {props.children}
+      <div className="ma__header_slim__header-container ma__container">
+        <div className="ma__header_slim__logo"><SiteLogo /></div>
```

Comment:
Is `SiteLogo` required? If so, it should be marked as such in propTypes. If not this will fail.

Reply:
added `.isRequired` in the proptype check